specification/frontdoor/resource-manager/readme.md

```diff
@@ -39,8 +39,25 @@ These settings apply only when `--tag=package-2022-05` is specified on the comma
 input-file:
   - Microsoft.Network/stable/2022-05-01/network.json
   - Microsoft.Network/stable/2022-05-01/webapplicationfirewall.json
-  - Microsoft.Network/stable/2020-05-01/frontdoor.json
+  - Microsoft.Network/stable/2021-06-01/frontdoor.json
+  - Microsoft.Network/stable/2019-11-01/networkexperiment.json
+```
+
+### Tag: package-2021-06
+
+These settings apply only when `--tag=package-2021-06` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-06'
+input-file:
+  - Microsoft.Network/stable/2021-06-01/frontdoor.json
+  - Microsoft.Network/stable/2021-06-01/webapplicationfirewall.json
+  - Microsoft.Network/stable/2021-06-01/network.json
   - Microsoft.Network/stable/2019-11-01/networkexperiment.json
+directive:
+  - where:
+      - $.paths
+    suppress:
+      - OperationsAPIImplementation
```

Comment:
since we are adding a new tag, instead of changing the old one (package-2021-05), could we here
- leave the old tag package-2021-05 unchanged
- make the tag on line 31 in this file to point your new tag package-2021-06
instead? 
This will trigger all the SDK to update according to the new things in this PR

Reply:
@ArcturusZhang Can you take a look at my response